src/main.js

```diff
@@ -233,15 +233,17 @@ function configureCommandlineSwitchesSync(cliArgs) {
 
 		// Append Electron flags to Electron
 		if (SUPPORTED_ELECTRON_SWITCHES.indexOf(argvKey) !== -1) {
-
-			if (
+			if (argvValue) {
```

Comment:
We should inverse the else branch `argvValue === true || argvValue === 'true'` with this branch, otherwise the new change will never run the else branch for truthy values.

Reply:
Nice catch. I missed that the next branch was an else if and not just a regular if